geth/jail/execution_policy.go

```diff
@@ -13,6 +13,89 @@ import (
 	"github.com/ethereum/go-ethereum/rpc"
 	"github.com/robertkrimen/otto"
 	"github.com/status-im/status-go/geth/common"
+	"github.com/status-im/status-go/geth/params"
+)
+
+// rpcRouteID defines a int type used for specifying command routing.
+type rpcRouteID int
+
+// local and upstream RouteID
+const (
+	localCommand rpcRouteID = iota + 1
+	upstreamCommand
+)
+
+// map of command routes
+var (
+	rpcCommandsRoute = map[string]rpcRouteID{
```

Comment:
Do we want to maintain this map? If a new Whisper command is added, we need to add it here. We could keep only a list of exceptions from the rules instead.